### PR TITLE
Adding the IOTranslator code to the branch

### PR DESCRIPTION
**Pull request by vtaneka**
Adding the IOtranslator code

### CODE REVIEW

File: server/controllers/filestore.controller.js [hunk 1]
```diff
@@ -3,15 +3,110 @@ const config = require('../config/config');
...
+    }
+
+    saveTaskRes(taskId, stepIdx, fileName, fileContent, callback){
+        
+        var absFilePath = this.getFileStoretaskFolderPath(taskId, stepIdx);
+
+        // var relativeXmlPath = this.getTaskFolderPath(taskId);
+        var relativeXmlPath = this.getSimsXmlTaskFolderPath(taskId, stepIdx);
+        
+        var srcPath  = config.fileStore.resourceFolder +  srcPath;
```
Comment (tushar-compro) [inline server/controllers/filestore.controller.js]: srcPath = x + srcPath ???

File: server/controllers/filestore.controller.js [hunk 2]
```diff
@@ -3,15 +3,110 @@ const config = require('../config/config');
...
 };
 
 class FileStoreController {
 
+    getTaskRes(filepath, callback){
+        this.readTaskRes(filepath, callback);
+    }
+
+    getFileStoretaskFolderPath(taskId, stepIdx){
+        return config.fileStore.xmlFolder + taskId + "/" + stepIdx + "/";
```
Comment (tushar-compro) [inline server/controllers/filestore.controller.js]: shouldn't this function be named getFileStoreStepFolderPath ???

File: server/libs/skills/common/baseSkill.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+          }
+          else{
+            callback(error);
+          }
+      })
+      
+    //   return paramValueObj["DocTitle"].path;
+    }
+    assignTopValue(skillParams, callback){
+
```
Comment (tushar-compro) [inline server/libs/skills/common/baseSkill.js]: fn name not appropriate

**Review by dhirendra-singh81 — APPROVED** [review]
Approving based on Tushar's review

**Review by tushar-compro — APPROVED** [review]

**Pull request merged**